try{
    print("trying")
    print(1+"A")
}
except NameError{
    print("NameError")
}
except Exception e{
    print(repr(e))
}